demo/actions/incident_notifications.py

```diff
@@ -0,0 +1,31 @@
+from response.core.models import Incident
+from response.slack.models import CommsChannel
+from response.slack.decorators import recurring_notification, single_notification
+
+@recurring_notification(interval_mins=5, max_notifications=5)
+def remind_severity(incident: Incident):
```

Comment:
The incident notifications you've added here for `remind_severity` and `remind_incident_lead` are part of the default code base so we don't need to duplicate them here as we'll inherit them from the Response pip package. Please can you remove these?

Reply:
Yep - removing the defaults.